lib/backend/k8s/thirdparty/global_config.go

```diff
@@ -0,0 +1,78 @@
+package thirdparty
+
+import (
+	"encoding/json"
+
+	"k8s.io/client-go/pkg/api"
+	"k8s.io/client-go/pkg/api/meta"
+	"k8s.io/client-go/pkg/api/unversioned"
+	"k8s.io/client-go/pkg/runtime/schema"
+)
+
+type GlobalConfigSpec struct {
+	Name  string `json:"name"`
+	Value string `json:"value"`
```

Comment:
Is it OK that Value is a string? Do we support GlobalConfig that isn't a string?